Makefile

```diff
@@ -212,7 +215,16 @@ $(TEST_EXE): debug
 test: $(TEST_EXE)
 	test x$(CROSS) = xOFF && $(TEST_EXE) || test x$(CROSS) = xON
 
+static-configure: $(LIBUV_PREFIX) $(LIBCURL_PREFIX)
+	(test x$(TOOLCHAIN) = x && $(CONFIG_CMD) -DCMAKE_BUILD_TYPE=Release -DSTATIC_LINK=ON -DRELEASE_MOTTO="$(shell cat motto.txt)" -DCMAKE_C_FLAGS='$(CFLAGS)' -DCMAKE_CXX_FLAGS='$(CXXFLAGS)' -DLIBUV_ROOT='$(LIBUV_PREFIX)' -DLIBCURL_ROOT='$(LIBCURL_PREFIX)' ) || (test x$(TOOLCHAIN) != x && $(CONFIG_CMD) -DCMAKE_BUILD_TYPE=Release -DSTATIC_LINK=ON -DRELEASE_MOTTO="$(shell cat motto.txt)" -DCMAKE_C_FLAGS='$(CFLAGS)' -DCMAKE_CXX_FLAGS='$(CXXFLAGS)' -DLIBUV_ROOT='$(LIBUV_PREFIX)' -DLIBCURL_ROOT='$(LIBCURL_PREFIX)' -DCMAKE_TOOLCHAIN_FILE=$(TOOLCHAIN) -DNATIVE_BUILD=OFF )
+
+static: static-configure
+	$(MAKE) -C '$(BUILD_ROOT)'
+	cp $(EXE) $(REPO)/lokinet-static
 
+$(LIBCURL_PREFIX):
+	mkdir -p $(BUILD_ROOT)
+	git clone -b "$(LIBCURL_VERSION)" https://github.com/curl/curl "$(LIBCURL_PREFIX)"
```

Comment:
It's probably quite a bit faster to download `https://github.com/curl/curl/releases/download/curl-7_67_0/curl-7.67.0.tar.xz`, which doesn't need to download the full repo with history but just the release code.